content/en/continuous_integration/guides/code_coverage.md

```diff
@@ -77,6 +77,103 @@ When code coverage is available, the Datadog Tracer (v2.31.0 or later) reports i
 
 If you are using [Coverlet][101] to compute your code coverage, indicate the path to the report file in the `DD_CIVISIBILITY_EXTERNAL_CODE_COVERAGE_PATH` environment variable when running `dd-trace`. The report file must be in the OpenCover or Cobertura formats. Alternatively, you can enable the Datadog Tracer’s built-in code coverage calculation with the `DD_CIVISIBILITY_CODE_COVERAGE_ENABLED=true` env variable.
 
+### Advanced options (Supported through .runsettings)
+
+The Datadog Tracer's built-in code coverage has support for both `Coverlet` and `VS Code Coverage` options through the `.runsettings` file.
+
+#### File structure
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+    <DataCollectionRunSettings>
+        <DataCollectors>
+            <DataCollector friendlyName="DatadogCoverage">
+                <Configuration>
+                    <!-- Datadog Code Coverage settings -->
+                    ...
+                </Configuration>
+            </DataCollector>
+        </DataCollectors>
+    </DataCollectionRunSettings>
+</RunSettings>
+```
+
+#### Coverlet options
+
+| Option                   | Summary                                                                                                                                                         |
+|:-------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ExcludeByAttribute       | Ignore methods, classes or assemblies decorated with attributes from code coverage.                                                                                                                |
+| ExcludeByFile            | Ignore specific source files from code coverage.                                                                                                                |
+| Exclude                  | Exclude from code coverage analysing using filter expressions.                                                                                                  |
+
+##### Attributes
```

Comment:
I think an example might be nice here but that's probably because I don't know .NET .